### ISSUE TITLE
Incompatible with company-coq

### ISSUE THREAD

**Issue by JasonGross** [psteckler/ProofGeneral#24]
This fork of PG seems incompatible with @cpitclaudel 's company-coq; I get `Symbol's value as variable is void: proof-shell-unicode`

**Comment by psteckler** [comment]
Yes, we'll have to port company-coq to this new code. I'm supposing there
will be changes at both ends. As I recall, there will only a few API calls
involved, so my expectation is that this effort won't be too difficult.

**Comment by cpitclaudel** [comment]
Hmm.  There's no easy way for me t distribute two separate copies of company-coq based on different releases of PG, so any changes that I'll do will have to be under an `if` checking which version of PG is running.  Ideally, we can tweak the new PG to minimize these issues :)

**Comment by psteckler** [comment]
PG/xml now supports company-coq about half-way. I've identified the remaining issues, which we'll look at soon.

**Comment by JasonGross** [comment]
Any update on this?  This is currently the biggest issue blocking me from using PG-xml more.

**Comment by psteckler** [comment]
There are a number of contact points between company-coq and PG that have yet to be resolved. I think Clement is looking at my list of these to see how we can proceed.

**Comment by cpitclaudel** [comment]
I am, but other things keep popping up.  I'm hopping to start looking at this this week.
